docs/src/explanations/connectable.md

```diff
@@ -309,7 +309,7 @@ This is the same error a user would get using a mono-directioned operator: `x :=
 *Whether a component is drivable is irrelevant to the semantics of any connection operator attempting to drive to it.*
 
 In summary, the port-direction computation is relative to the root marked `IO`, but connection-direction computation is relative to the consumer/producer that the connection is doing.
-This has the positive property that connection semantics are solely based on the Chisel structural type and its relative alignments of the consumer/producer (nothing more, nothing less).
+This has the positive property that connection semantics are solely based on the Chisel structural type and the relative alignments of the consumer and producer (nothing more, nothing less).
```

Comment:
hmm this might be subtly changing the meaning of the sentence which might have not been that clear before to something further from what we want it to say...

Reply:
@mwachs5 just getting back to this after getting the CLA signed, can you say more about what the intended meaning behind this sentence is? Looking back on this now, I have no problem with reverting the change on this sentence, if that's the best solution.

Reply:
I just went ahead and removed this change.